subfunctions/demo_CaBBI_QGIF

```diff
@@ -0,0 +1,232 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
I removed these remarks from all files; please let me know if you meant something else.